### DISCUSSION TITLE
Medusa

### DISCUSSION THREAD

**Discussion by ScriptedAlchemy** [module-federation/module-federation-examples#612] [Show and tell]
Medusa - previously known as the Federation Dashboard. 

Medusa will act as a UI to the federated webpack graph. You are about to make changes to federated apps at runtime, without redeploying. 

Code freezing parts of the graph, instant rollbacks, ab tests, multiple versioned, and nested versioned remote modules. 

Here's a sneak peek. 
There are 5 federated applications on the page, and one is a DSL design system library used by the various apps. One version is red, and one version is blue. 

https://youtu.be/96_TMYue6Dk

**Comment by XiaofengXie16** [comment]
I'm curious if there is any update on this? (I know you tweeted about publishing a new video on medusa)

**Reply by brriann** [reply]
Also interested in this.  Will Medusa be open source, with contributions welcomed?  If so, would be interested to contribute as possible.
